### PR TITLE
Fix some NameErrors, don't delete unused [...]

### PR DESCRIPTION
**Pull request by jbrockmendel**
Non-deletion part of #4852.

### CODE REVIEW

File: statsmodels/miscmodels/count.py [hunk 1]
```diff
@@ -250,88 +250,3 @@ def nloglikeobs(self, params):
...
-    np.random.seed(98765678)
-    nobs = 1000
-    rvs = np.random.randn(nobs,6)
-    data_exog = rvs
-    data_exog = sm.add_constant(data_exog, prepend=False)
-    xbeta = 1 + 0.1*rvs.sum(1)
-    data_endog = np.random.poisson(np.exp(xbeta))
-    #print(data_endog
-
-    modp = MyPoisson(data_endog, data_exog)
```
Comment (jbrockmendel) [inline statsmodels/miscmodels/count.py]: There is no `MyPoisson`.  Everything here is unusable.

File: statsmodels/sandbox/archive/linalg_covmat.py [hunk 2]
```diff
@@ -77,15 +77,6 @@ def logpdf(self, x):
 
 from .linalg_decomp_1 import SvdArray, OneTimeProperty
 
-class MultivariateNormal(object):
```
Comment (jbrockmendel) [inline statsmodels/sandbox/archive/linalg_covmat.py]: There is a not-broken MultivariateNormal class defined below.

**Comment by jbrockmendel** [comment]
@TomAugspurger little help?  NameErrors should be high on the priority list here

**Comment by josef-pkt** [comment]
commit and merge message need to include the prefix for the type of changes, e.g. MAINT

**Comment by TomAugspurger** [comment]
Will do. Does Github have any way to enforce that?

**Comment by josef-pkt** [comment]
There is possibility to enforce it AFAIK, except reminders by one of the maintainers 
(e.g. Skipper needed to remind me several times until I didn't forget anymore to edit the merge message)

**Pull request merged**